recipes/ibis-framework/meta.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: ibis-framework
+  version: "0.7.1"
+
+
+source:
+  git_url: https://github.com/cloudera/ibis
+  git_rev: v0.7.1
+
+build:
+  script: python setup.py install
```

Comment:
Since this package uses setuptools you should install it using `script: python setup.py install --single-version-externally-managed --record record.txt`. That makes the package easily uninstallable.